files/pull-sandbox-image.sh

```diff
@@ -8,4 +8,26 @@ if [[ "$(sudo ctr --namespace k8s.io image ls | grep $sandbox_image)" != "" ]];
   exit 0
 fi
 
-/etc/eks/containerd/pull-image.sh "${sandbox_image}"
+MAX_RETRIES=3
+
+function retry() {
+  local rc=0
+  for attempt in $(seq 0 $MAX_RETRIES); do
+    rc=0
+    [[ $attempt -gt 0 ]] && echo "Attempt $attempt of $MAX_RETRIES" 1>&2
+    "$@"
+    rc=$?
+    [[ $rc -eq 0 ]] && break
+    [[ $attempt -eq $MAX_RETRIES ]] && exit $rc
+    local jitter=$((1 + RANDOM % 10))
+    local sleep_sec="$(($((5 << $((1 + $attempt)))) + $jitter))"
+    sleep $sleep_sec
+  done
+}
+
+ecr_password=$(retry aws ecr get-login-password)
+if [[ -z ${ecr_password} ]]; then
+  echo >&2 "Unable to retrieve the ECR password."
+  exit 1
+fi
+sudo retry crictl pull "${sandbox_image}" --creds AWS:${ecr_password}
```

Comment:
I just checked my node and crictl wasn't there. I think you need to install it as well.

Reply:
Right, missed in my commit 😓 . We mulled over adding `cri-tools` in the past because we aren't the maintainers of that package in AL. We don't have much of a choice now; the only other option would be manually labelling the image as pinned after we pull it with `ctr`.